include/seastar/http/request.hh

```diff
@@ -65,7 +65,7 @@ struct request {
     mutable size_t _bytes_written = 0;
     std::unordered_map<sstring, sstring, seastar::internal::case_insensitive_hash, seastar::internal::case_insensitive_cmp> _headers;
     std::unordered_map<sstring, sstring> query_parameters;
-    httpd::parameters param;
+    httpd::parameters param; // deprecated: it is used to store last value of query parameters, but will be removed in the future
```

Comment:
Huh? Why param is deprecated, but not the query_parameters?
Also -- comment is not enough, mind using the `[[deprecated("reason")]]` decoration